src/bots/filler.ts

```diff
@@ -1581,7 +1595,11 @@ export class FillerBot implements Bot {
 				);
 				return;
 			}
-			this.sendFillTx(fillTxId, [nodeToFill], tx);
+			if (!this.dryRun && !simError) {
```

Comment:
should probably log somewhere that we skipped from sim error

Reply:
also if we see simError, can we call handleTransactionLogs? i feel like otherwise we might not filter out the mms w too much leverage ...